README.md

```diff
@@ -168,6 +168,50 @@ entries, that is:
 - `${rock:which:cmd}` expands to the full path to the command `cmd` within
    the autoproj workspace's PATH.
 
+### Syskit Workflow
+
+The only syskit-specific workflow available right now is the ability to
+configure the Syskit IDE to open file links directly in VSCode. To do so, open
+`$HOME/.config/syskit.conf` and create or update the `[Main]` section to have
+
+~~~
+cmdline=code --goto %FILEPATH:%LINENO
+~~~
+
+For instance, a freshly created `syskit.conf` would look like:
+
+~~~
+[Main]
+cmdline=code --goto %FILEPATH:%LINENO
+~~~
+
+Moreover, as part of a good general Syskit workflow in VSCode, we recommend
+creating task entries to start `rock-gazebo` and the IDE. For instance:
```

Comment:
Shouldn't the task provider generate those?

Reply:
Possibly, but it's work I don't have time for right now, so I thought documenting it would already be better than nothing (we need to list and handle scenes and robots, so it's not completely straightforward)

Reply:
Ok. I will do it. I have added that to my vscode-rock TODO list (along with installation-manifest watcher and add package to workspace command). In the mean time, I guess we should merge this :+1: